packages/web-components/src/components/va-table/va-table-inner/va-table-row.tsx

```diff
@@ -8,7 +8,7 @@ import { Component, Host, h } from '@stencil/core';
 export class VaTableRow {
   render() {
     return (
-      <Host role="row">
+      <Host>
```

Comment:
I was getting an Axe error when this had `role="row"`. It looks like this was [flagged originally](https://github.com/department-of-veterans-affairs/component-library/pull/1128#pullrequestreview-2020004769) and it was [removed in that PR](https://github.com/department-of-veterans-affairs/component-library/pull/1128/commits/bff1cdf3369b3bb6706556a8a108aff5075b32ad). But some how made it back in somewhere somehow. 🤔 I think it should be removed based on Ryan's comments in the original PR.

![Screenshot 2024-11-19 at 9 36 18 AM](https://github.com/user-attachments/assets/021d357b-dd83-4b07-8983-c5230d51f2c6)